main.cpp

```diff
@@ -56,10 +53,27 @@ void variableDeclarations()
 {
     //example:
     int number = 2; //declaration of a primitive named 'number' with an initial value of '2'
+     int beats = 4;
+     int subdivisions = 16;
+     int tempo = 120;
+
+     char tube = 'x';
+     char head = 'a';
+     char tuning = 'c';
+
+     bool bypassed = true;
+     bool stereo = false;
+     bool saveInPresent = true;
+
+     float bass = 1.8f;
+     float treble = 3.3f;
+     float master = 11.f;
+
+     double cutoff = 200.23;
+     double q = 4.33;
+     double gain = 6.66; 
     
-    
-    
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    ignoreUnused(number,beats,subdivisions,tempo,tube,head,tuning,bypassed,stereo,saveInPresent,bass,treble,master,cutoff,q,gain); //passing each variable declared to the ignoreUnused() function
```

Comment:
Please correct the indentation.  the coding style guide in this course is in the Readme.md
- [ ] revise

Reply:
Indentations are now corrected